src/taipy/rest/api/resources/pipeline.py

```diff
@@ -266,9 +270,7 @@ def __init__(self, **kwargs):
 
     @_middleware
     def post(self, pipeline_id):
-        try:
-            manager = _PipelineManagerFactory._build_manager()
-            manager._submit(pipeline_id)
-            return {"message": f"Executed pipeline {pipeline_id}"}
-        except NonExistingPipeline:
-            return make_response(jsonify({"message": f"Pipeline {pipeline_id} not found"}), 404)
+        _get_or_raise(pipeline_id)
+        manager = _PipelineManagerFactory._build_manager()
+        manager._submit(pipeline_id)
+        return {"message": f"Executed pipeline {pipeline_id}"}
```

Comment:
Same here:

Pipeline {} was executed.

(or submitted? what is the difference? asynchronism I suspect)